typings/css-properties.d.ts

```diff
@@ -1525,5 +1525,5 @@ export interface CSSProperties {
   '::first-line'?: CSSProperties
   '::selection'?: CSSProperties
 
-  [propertyName: string]: string | number | CSSProperties | undefined
+  [propertyName: string]: string | number | boolean | Function | CSSProperties | undefined
```

Comment:
Could you provide an example where this would be useful 🙂? I'm not sure whether it is really needed